cmd/terramate/cli/script_info.go

```diff
@@ -15,7 +18,9 @@ import (
 )
 
 func (c *cli) printScriptInfo() {
-	labels := c.parsedArgs.Script.Info.Labels
+	c.checkScriptEnabled()
```

Comment:
nitpick:
What do you think about moving this tn the top level switch case (in cmd/terramate/cli/cli.go) before `c.printScriptInfo()` is invoked? (similar to how we use `c.setupGit()`)

Reply:
It makes sense. I was on the fence if this or placing per command made sense. I updated.